scripts/st2bootstrap-deb.sh

```diff
@@ -1,145 +1,165 @@
+  st2 auth test -p Ch@ngeMe
+  # A shortcut to authenticate and export the token
+  export ST2_AUTH_TOKEN=$(st2 auth test -p Ch@ngeMe -t)
 
-}
+  # List the actions from a 'core' pack
+  st2 action list --pack=core
 
-install_webui_and_setup_ssl_termination() {
-	# Install st2web and nginx
-	sudo apt-get install -y st2web nginx
+  # Run a local shell command
+  st2 run core.local -- date -R
 
-	# Generate self-signed certificate or place your existing certificate under /etc/ssl/st2
-	sudo mkdir -p /etc/ssl/st2
-	sudo openssl req -x509 -newkey rsa:2048 -keyout /etc/ssl/st2/st2.key -out /etc/ssl/st2/st2.crt \
-	-days XXX -nodes -subj "/C=US/ST=California/L=Palo Alto/O=StackStorm/OU=Information \
-	Technology/CN=$(hostname)"
+  # See the execution results
+  st2 execution list
 
-	# Remove default site, if present
-	sudo rm /etc/nginx/sites-enabled/default
-	# Copy and enable StackStorm's supplied config file
-	sudo cp /usr/share/doc/st2/conf/nginx/st2.conf /etc/nginx/sites-available/
-	sudo ln -s /etc/nginx/sites-available/st2.conf /etc/nginx/sites-enabled/st2.conf
+  # Fire a remote comand via SSH (Requires passwordless SSH)
+  st2 run core.remote hosts='127.0.0.1' -- uname -a
 
-	sudo service nginx restart
+  # Install a pack
+  st2 run packs.install packs=st2
 }
 
 ok_message() {
-    ST2_IP=`ifconfig | grep 'inet addr' | awk '{print $2 }' | awk 'BEGIN {FS=":"}; {print $2}' | head -n 1`
-
-    echo ""
-    echo ""
-    echo "███████╗████████╗██████╗      ██████╗ ██╗  ██╗";
-    echo "██╔════╝╚══██╔══╝╚════██╗    ██╔═══██╗██║ ██╔╝";
-    echo "███████╗   ██║    █████╔╝    ██║   ██║█████╔╝ ";
-    echo "╚════██║   ██║   ██╔═══╝     ██║   ██║██╔═██╗ ";
-    echo "███████║   ██║   ███████╗    ╚██████╔╝██║  ██╗";
-    echo "╚══════╝   ╚═╝   ╚══════╝     ╚═════╝ ╚═╝  ╚═╝";
-    echo ""
-    echo "  st2 is installed and ready to use."
-    echo ""
-    echo "Head to https://${ST2_IP} to access the WebUI"
-    echo ""
-    echo "Don't forget to dive into our documentation! Here are some resources"
-    echo "for you:"
-    echo ""
-    echo "* Documentation  - https://docs.stackstorm.com"
-    echo "* Knowledge Base - https://stackstorm.reamaze.com"
-    echo ""
-    echo "Thanks for installing StackStorm! Come visit us in our Slack Channel"
-    echo "and tell us how it's going. We'd love to hear from you!"
-    echo "http://stackstorm.com/community-signup"
+  ST2_IP=`ifconfig | grep 'inet addr' | awk '{print $2 }' | awk 'BEGIN {FS=":"}; {print $2}' | head -n 1`
+
+  echo ""
+  echo ""
+  echo "███████╗████████╗██████╗      ██████╗ ██╗  ██╗";
+  echo "██╔════╝╚══██╔══╝╚════██╗    ██╔═══██╗██║ ██╔╝";
+  echo "███████╗   ██║    █████╔╝    ██║   ██║█████╔╝ ";
+  echo "╚════██║   ██║   ██╔═══╝     ██║   ██║██╔═██╗ ";
+  echo "███████║   ██║   ███████╗    ╚██████╔╝██║  ██╗";
+  echo "╚══════╝   ╚═╝   ╚══════╝     ╚═════╝ ╚═╝  ╚═╝";
+  echo ""
+  echo "  st2 is installed and ready to use."
+  echo ""
+  echo "Head to https://${ST2_IP} to access the WebUI"
```

Comment:
Heh. I didn't even realize that the message had this. I wanted the banner. I'll remove this particular line. I agree it is confusing.